packages/trader/src/Modules/Reports/Containers/open-positions.jsx

```diff
@@ -312,13 +313,17 @@ const OpenPositions = ({
         // eslint-disable-next-line react-hooks/exhaustive-deps
     }, []);
 
+    React.useEffect(() => {
+        setLoginId(loginid);
+    }, [loginid]);
+
     React.useEffect(() => {
         checkForMultiplierContract(previous_active_positions);
         // eslint-disable-next-line react-hooks/exhaustive-deps
     }, [previous_active_positions]);
 
     const checkForMultiplierContract = (prev_active_positions = []) => {
-        if (!has_multiplier_contract && active_positions !== prev_active_positions) {
+        if ((!has_multiplier_contract && active_positions) || (loginid !== loginId_saved) !== prev_active_positions) {
```

Comment:
`(loginid !== loginId_saved) !== prev_active_positions` 
This condition looks incomplete 🤔

Reply:
@maryiafrantsava-deriv I think the condition is added in the wrong place. It should be
```
 if ((!has_multiplier_contract && active_positions !== prev_active_positions) || (loginid !== login_id_saved) ) {
 ```